--- filedlgbrowser.orig.c	2021-03-10 19:44:28.000000000 +0100
+++ filedlgbrowser.c	2021-03-10 19:44:45.000000000 +0100
@@ -19,13 +19,13 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
+#define COBJMACROS
+#define NONAMELESSUNION
+
 #include <stdarg.h>
 #include <stdio.h>
 #include <string.h>
 
-#define COBJMACROS
-#define NONAMELESSUNION
-
 #include "windef.h"
 #include "winbase.h"
 #include "winnls.h"
