--- Makefile.orig.in	2021-03-11 11:15:28.000000000 +0100
+++ Makefile.in	2021-03-11 11:15:31.000000000 +0100
@@ -1,7 +1,7 @@
 EXTRADEFS = -DD3DX_SDK_VERSION=36
 MODULE    = d3dx9_36.dll
 IMPORTLIB = d3dx9
-IMPORTS   = d3d9 d3dcompiler dxguid d3dxof ole32 gdi32 user32 ucrtbase
+IMPORTS   = d3d9 d3dcompiler dxguid d3dxof ole32 gdi32 user32 ucrtbase msvcrt
 DELAYIMPORTS = windowscodecs
 
 EXTRADLLFLAGS = -mno-cygwin
