--- sync.orig.c	2021-03-11 10:18:02.000000000 +0100
+++ sync.c	2021-03-11 13:50:13.000000000 +0100
@@ -18,13 +18,14 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
+#define NONAMELESSUNION
+
 #include <stdarg.h>
 #include <string.h>
 #include <stdio.h>
 
 #include "ntstatus.h"
 #define WIN32_NO_STATUS
-#define NONAMELESSUNION
 #include "windef.h"
 #include "winbase.h"
 #include "wincon.h"
@@ -1436,7 +1437,7 @@
     return !RtlRunOnceExecuteOnce( once, (PRTL_RUN_ONCE_INIT_FN)func, param, context );
 }
 
-#ifdef __i386__
+#if defined(__i386__)
 
 /***********************************************************************
  *	     InterlockedCompareExchange   (kernelbase.@)
@@ -1485,5 +1486,4 @@
                   "lock; xaddl %eax,(%edx)\n\t"
                   "decl %eax\n\t"
                   "ret $4")
-
 #endif  /* __i386__ */
