--- debug.orig.c	2021-03-11 10:26:14.000000000 +0100
+++ debug.c	2021-03-11 10:27:48.000000000 +0100
@@ -128,7 +128,7 @@
 /***********************************************************************
  *           DebugBreak   (kernelbase.@)
  */
-#if defined(__i386__) || defined(__x86_64__)
+#if defined(__i386__) || defined(__x86_64__) && !defined(__i386_on_x86_64__)
 __ASM_STDCALL_FUNC( DebugBreak, 0, "jmp " __ASM_STDCALL("DbgBreakPoint", 0) )
 #else
 void WINAPI DebugBreak(void)
