--- console.orig.c	2021-03-11 10:14:39.000000000 +0100
+++ console.c	2021-03-11 10:14:55.000000000 +0100
@@ -577,7 +577,7 @@
 /***********************************************************************
  *	GetLargestConsoleWindowSize   (kernelbase.@)
  */
-#if defined(__i386__) && !defined(__MINGW32__)
+#if (defined(__i386__) || defined(__i386_on_x86_64__)) && !defined(__MINGW32__)
 #undef GetLargestConsoleWindowSize
 DWORD WINAPI DECLSPEC_HOTPATCH GetLargestConsoleWindowSize( HANDLE handle )
 {
