--- main.orig.c	2021-03-11 11:31:12.000000000 +0100
+++ main.c	2021-03-11 11:31:27.000000000 +0100
@@ -16,13 +16,13 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
+#define COBJMACROS
+#define NONAMELESSUNION
+
 #include <stdarg.h>
 #include <string.h>
 #include <math.h>
 
-#define COBJMACROS
-#define NONAMELESSUNION
-
 #include "windef.h"
 #include "winbase.h"
 #include "winuser.h"
