--- instr.orig.c	2021-03-11 14:52:27.000000000 +0100
+++ instr.c	2021-03-11 14:55:18.000000000 +0100
@@ -33,7 +33,7 @@
 #include "wine/debug.h"
 #include "wine/exception.h"
 
-#ifdef __i386__
+#if defined(__i386__) || defined(__i386_on_x86_64__)
 
 WINE_DEFAULT_DEBUG_CHANNEL(int);
 
@@ -50,7 +50,7 @@
 static inline struct idtr get_idtr(void)
 {
     struct idtr ret;
-#ifdef __GNUC__
+#if defined(__GNUC__) && !defined(__i386_on_x86_64__)
     __asm__( "sidtl %0" : "=m" (ret) );
 #else
     ret.base = (BYTE *)idt;
