--- stg_prop.orig.c	2021-03-11 15:06:04.000000000 +0100
+++ stg_prop.c	2021-03-11 15:07:13.000000000 +0100
@@ -37,15 +37,15 @@
  *   PropertyStorage_ReadFromStream
  */
 
+#define COBJMACROS
+#define NONAMELESSUNION
+
 #include <assert.h>
 #include <stdarg.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
-#define COBJMACROS
-#define NONAMELESSUNION
-
 #include "windef.h"
 #include "winbase.h"
 #include "winnls.h"
