--- filelockbytes.orig.c	2021-03-11 15:05:03.000000000 +0100
+++ filelockbytes.c	2021-03-11 15:07:40.000000000 +0100
@@ -20,6 +20,10 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
+#define COBJMACROS
+#define NONAMELESSUNION
+#define NONAMELESSSTRUCT
+
 #include <assert.h>
 #include <stdlib.h>
 #include <stdarg.h>
@@ -27,10 +31,6 @@
 #include <string.h>
 #include <limits.h>
 
-#define COBJMACROS
-#define NONAMELESSUNION
-#define NONAMELESSSTRUCT
-
 #include "windef.h"
 #include "winbase.h"
 #include "winuser.h"
