--- usrmarshal.orig.c	2021-03-11 15:06:11.000000000 +0100
+++ usrmarshal.c	2021-03-11 15:07:08.000000000 +0100
@@ -18,13 +18,13 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
+#define COBJMACROS
+#define NONAMELESSUNION
+
 #include <stdio.h>
 #include <stdarg.h>
 #include <string.h>
 
-#define COBJMACROS
-#define NONAMELESSUNION
-
 #include "windef.h"
 #include "winbase.h"
 #include "wingdi.h"
