--- authorization.m.bak	2021-03-10 13:29:03.000000000 +0100
+++ authorization.m	2021-03-10 13:47:37.000000000 +0100
@@ -37,9 +37,6 @@
 
 #endif
 
-static NSString* const WineWillShowPermissionDialogNotification = @"WineWillShowPermissionDialogNotification";
-static NSString* const WineDidShowPermissionDialogNotification = @"WineDidShowPermissionDialogNotification";
-
 /***********************************************************************
  *              CoreAudio_get_capture_authorization_status
  *
@@ -78,8 +75,6 @@
 
     if ([AVCaptureDevice respondsToSelector:@selector(requestAccessForMediaType:completionHandler:)])
     {
-        [[NSNotificationCenter defaultCenter] postNotificationName:WineWillShowPermissionDialogNotification
-                                                            object:nil];
         semaphore = dispatch_semaphore_create(0);
         [AVCaptureDevice requestAccessForMediaType:AVMediaTypeAudio
                                  completionHandler:^(BOOL granted){
@@ -89,8 +84,6 @@
 
         dispatch_semaphore_wait(semaphore, DISPATCH_TIME_FOREVER);
         dispatch_release(semaphore);
-        [[NSNotificationCenter defaultCenter] postNotificationName:WineDidShowPermissionDialogNotification
-                                                            object:nil];
     }
 
     [pool release];
