--- vulkan_private.orig.h	2021-03-16 13:36:27.000000000 +0100
+++ vulkan_private.h	2021-03-16 13:53:00.000000000 +0100
@@ -21,7 +21,7 @@
 #define __WINE_VULKAN_PRIVATE_H
 
 /* Perform vulkan struct conversion on 32-bit x86 platforms. */
-#if defined(__i386__)
+#if defined(__i386__) || defined(__i386_on_x86_64__)
 #define USE_STRUCT_CONVERSION
 #endif
 
@@ -31,6 +31,7 @@
 #define VK_NO_PROTOTYPES
 #include "wine/vulkan.h"
 #include "wine/vulkan_driver.h"
+#include <wine/32on64utils.h>
 
 #include "vulkan_thunks.h"
 
@@ -122,7 +123,7 @@
 
 static inline struct wine_cmd_pool *wine_cmd_pool_from_handle(VkCommandPool handle)
 {
-    return (struct wine_cmd_pool *)(uintptr_t)handle;
+    return TRUNCCAST(struct wine_cmd_pool *, (uintptr_t)handle);
 }
 
 static inline VkCommandPool wine_cmd_pool_to_handle(struct wine_cmd_pool *cmd_pool)
