--- treeview.orig.c	2021-03-10 18:30:43.000000000 +0100
+++ treeview.c	2021-03-10 18:30:56.000000000 +0100
@@ -40,6 +40,8 @@
  *   Scroll (instead of repaint) as much as possible.
  */
 
+#define NONAMELESSUNION
+
 #include <assert.h>
 #include <ctype.h>
 #include <stdarg.h>
@@ -47,8 +49,6 @@
 #include <limits.h>
 #include <stdlib.h>
 
-#define NONAMELESSUNION
-
 #include "windef.h"
 #include "winbase.h"
 #include "wingdi.h"
