--- main.orig.c	2021-03-10 20:13:46.000000000 +0100
+++ main.c	2021-03-10 20:14:02.000000000 +0100
@@ -16,12 +16,12 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
-#include <stdarg.h>
-#include <wchar.h>
-
 #define COBJMACROS
 #define NONAMELESSUNION
 
+#include <stdarg.h>
+#include <wchar.h>
+
 #include "windef.h"
 #include "winbase.h"
 #include "winnls.h"
