*** d3d11_include.h.orig	Mon Dec  6 18:44:58 2021
--- d3d11_include.h	Mon Dec  6 22:08:06 2021
***************
*** 25,30 ****
--- 25,31 ----
  // directly, although others from the same header work.
  // Some structures are missing from the mingw headers.
  #ifndef _MSC_VER
+ #if !defined(__MINGW64_VERSION_MAJOR) || __MINGW64_VERSION_MAJOR < 9
  typedef enum D3D11_FORMAT_SUPPORT2 { 
    D3D11_FORMAT_SUPPORT2_UAV_ATOMIC_ADD                                = 0x1,
    D3D11_FORMAT_SUPPORT2_UAV_ATOMIC_BITWISE_OPS                        = 0x2,
*************** typedef enum D3D11_FORMAT_SUPPORT2 { 
*** 41,46 ****
--- 42,48 ----
  } D3D11_FORMAT_SUPPORT2;
  #define D3D11_RESOURCE_MISC_TILE_POOL (0x20000)
  #define D3D11_RESOURCE_MISC_TILED     (0x40000)
+ #endif // !defined(__MINGW64_VERSION_MAJOR) || __MINGW64_VERSION_MAJOR < 9
  #ifndef __WINE__
  
  //MinGW-Headers supports these typedefs since 6.0.0
