*** fetchDependencies.orig	Thu Jun  2 13:15:33 2022
--- fetchDependencies	Thu Jun  2 13:14:37 2022
*************** else
*** 333,341 ****
  	update_repo ${REPO_NAME} ${REPO_URL} ${REPO_REV}
  
  	cd ${REPO_NAME}
! 	./update_glslang_sources.py
  	rm -rf build
! 	./build_info.py .        \
  		-i build_info.h.tmpl  \
  		-o build/include/glslang/build_info.h
  	cd -  > /dev/null
--- 333,341 ----
  	update_repo ${REPO_NAME} ${REPO_URL} ${REPO_REV}
  
  	cd ${REPO_NAME}
! 	python3 ./update_glslang_sources.py
  	rm -rf build
! 	python3 ./build_info.py .        \
  		-i build_info.h.tmpl  \
  		-o build/include/glslang/build_info.h
  	cd -  > /dev/null
