--- stdio.orig.h	2021-03-10 18:23:52.000000000 +0100
+++ stdio.h	2021-03-10 18:24:16.000000000 +0100
@@ -129,7 +129,15 @@
 int    __cdecl _vscprintf(const char*,__ms_va_list);
 int    __cdecl _vsnprintf(char*,size_t,const char*,__ms_va_list);
 #ifdef __i386_on_x86_64__
-int    __cdecl _vsnprintf(char* HOSTPTR,size_t,const char* HOSTPTR,__ms_va_list) __attribute__((overloadable)) asm(__ASM_NAME("wine__vsnprintf_HOSTPTR"));
+#if defined(__APPLE__) || (defined(_WIN32) && (defined(__i386__) || defined(__i386_on_x86_64__)))
+# define __ASM_NAME_PATCH(name) "_" name
+#else
+# define __ASM_NAME_PATCH(name) name
+#endif
+
+int    __cdecl _vsnprintf(char* HOSTPTR,size_t,const char* HOSTPTR,__ms_va_list) __attribute__((overloadable)) asm(__ASM_NAME_PATCH("wine__vsnprintf_HOSTPTR"));
+
+#undef __ASM_NAME_PATCH
 #endif
 int    __cdecl _vsnprintf_s(char*,size_t,size_t,const char*,__ms_va_list);
 int    __cdecl _vsprintf_p_l(char*,size_t,const char*,_locale_t,__ms_va_list);
